check_gpt/app.py

```diff
@@ -9,54 +9,66 @@
 # calls ChatGPT API with given prompt.
 def get_chatgpt_response(prompt: str) -> str:
     
-    short_messages = [
-            {"role": "system", "content": "You are going to write the opposite of the current text."},
-            {"role": "user", "content": prompt}
-        ]
-    
-    long_messages = [
-            {"role": "system", "content": "You are going to help me detect if the current text is generated by an AI model by rewriting the text"},
-            {"role": "user", "content": prompt}
-        ]
-    
-    model_prompt = []
-    
-    if len(prompt) < 280: 
-        model_prompt = short_messages
-    else: 
-        model_prompt = long_messages
-    
     
     completion = client.chat.completions.create(
-        model="gpt-4o",
-        messages=model_prompt
+    model="gpt-4o",
+    messages=[
+        {"role": "system", "content": "Rewrite the text in the reverse only the meaning."},
+        {
+            "role": "user",
+            "content": prompt
+        }]
     )
     return completion.choices[0].message.content
 
 # Compute Levenshtein ratio for edit distance
 def levenshtein_ratio(text1: str, text2:str) -> float:
     """
-    A result closer to 1: less edits made
-    A result closer to 0: more edits made
+    A result closer to 1: results are more similar
+    A result closer to 0: results are more different
     """
     
     if len(text1) == 0 and len(text2) == 0:
         return 1.0
     
     distance = Levenshtein.distance(text1, text2)
     max_len = max(len(text1), len(text2))
-    ratio = 1 - (distance / max_len)
+    ratio = (distance / max_len)
     
     return ratio
 
 
+
+def rewrite_text(original_text:str): 
+    """
+    Executes the rewrite using the equivalence method. 
+    """
+    first_rewrite = get_chatgpt_response(original_text)
+    paraphrased_text = get_chatgpt_response(first_rewrite)
+
+    return paraphrased_text
+
+
 # Function to analyze texts based on Levenshtein ratio
 def analyze_texts(original_text: str, paraphrased_text: str, ratio_threshold=0.6)-> bool:
     """
     Will return True if generated by ChatGPT, False otherwise
     """
-    ratio = levenshtein_ratio(original_text, paraphrased_text)
-    return True if ratio < ratio_threshold else False
+    
+    number_rewrites = 2
+    
+    rewrites = [rewrite_text(original_text) for i in range(number_rewrites)]
```

Comment:
I believe both entries in `rewrites` are the result of calling `get_chatgpt_response` twice, correct (and not 2x and 4x respectively)? Is this intended?